settings.gradle.kts

```diff
@@ -2,8 +2,8 @@ rootProject.name = "cloud-storage-connectors-for-apache-kafka"
 
 val assertJVersion by extra("3.26.0")
 val avroVersion by extra("1.11.3")
-val avroConverterVersion by extra("7.2.2")
-val avroDataVersion by extra("7.2.2")
+val avroConverterVersion by extra("7.6.0")
```

Comment:
As above for preferring the most recent patch version (in this case 7.6.2).

Reply:
I updated it to 7.7.1 instead to keep it aligned with all io confluent versions. wdyt ?